### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Signing into Azure Storage_Toggle Explorer Pane - Tree View_ScreenReader: When the voiceover focus lands on local and attached dropdown item that contains child items the screen reader announces 0 items.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/AzureStorageExplorer#5762]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_AzureToolsStorageExplorer_May2022;#Azure Tools Storage Explorer;#MAC;#FTP;#DesktopApp;#WCAG1.3.1;#Voiceover;#AINotInScope;#Regressed:06-17-22;#External:VoiceOver;#A11ycaioh;#Closed;

## Environment Details:
Application Name: Azure Tools Storage Explorer 
Application Version: 1.24.0
Mac Version: macOS Monterey 12.3.1
Screen Reader: Voiceover

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch Azure Tools Storage Explorer.
3. Sign in to Azure.
4. Tab to 'Explorer' pane tree view.
5. Tab to any of the 'Storage Accounts' that contains items
6. Turn on voiceover and observe the announcement.

## Actual:
When the voiceover focus lands on local and attached dropdown item that contains child items the screen reader announces 0 items.

## Expected:
When the voiceover focus lands on local and attached dropdown item that contains child items the screen reader should announce the number of items enclosed. For example "ABC, expanded, 3 items enclosed".

## User Impact:
Screen reader users will face difficulty in knowing the number of items enclosed within the parent control and may miss the content inside it.

## Attachment:
![3_ZeroItemsenclosed](https://user-images.githubusercontent.com/98393966/169968439-0b3cfd02-41e1-4438-b0d2-0bc7a391fb70.gif)

**Comment by MRayermannMSFT** [comment]
@bbarik06 we cannot reproduce this. Can you please elaborate on what key presses you used to get into this state?

**Comment by bbarik06** [comment]
@MRayermannMSFT First collapse all the tree view > bring focus to 'Refresh All' button and tab and use (Shift + Control + Option + Down arrow key) > Open 'Local & Attached' tree > Now move back and forth screen reader focus between the contents to see it announcing as 'Zero items enclosed'

**Comment by MRayermannMSFT** [comment]
@bbarik06 can you provide a full, end to end, screen recording of your reproducing this? Thanks.

**Comment by bbarik06** [comment]
@MRayermannMSFT Please find the full attachment.

![5762_New](https://user-images.githubusercontent.com/98393966/174248802-ab8d35e3-dcd7-421f-907e-0fc14b255c8c.gif)

**Comment by craxal** [comment]
Spoke with Ryan Shugart and Scott O'Hara at Accessibility office hours. This is due to the fact that Apple and VoiceOver don't support tree semantics; trees simply don't exist in Apple systems. Therefore, there's nothing we can do to fix this.

**Comment by bbarik06** [comment]
Closing this bug as per above comment received.
